### PR TITLE
Fix Twitter handles in Example project

### PR DESCRIPTION
**Pull request by BalestraPatrick**
## Changes in this pull request

I was exploring the example project when I noticed that two Twitter handles were wrong.
This will probably be the most useless PR ever in `IGListKit`, but it's good that you guys get the correct acknowledgment for your hard work! 🤓

## Pull request checklist

- [x] All tests pass. Demo project builds and runs.
- [x] I added tests, an experiment, or detailed why my change isn't tested.
- [x] I have reviewed the [contributing guide](https://github.com/Instagram/IGListKit/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by rnystrom** [comment]
Omg, sorry @jessesquires and @ryanolsonk 😂

**Comment by jessesquires** [comment]
lol

File: Example/IGListKitExamples/ViewControllers/MixedDataViewController.swift [hunk 1]
```diff
@@ -25,12 +25,12 @@ class MixedDataViewController: UIViewController, IGListAdapterDataSource {
...
         "Maecenas faucibus mollis interdum. Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit.",
         GridItem(color: UIColor(red: 237/255.0, green: 73/255.0, blue: 86/255.0, alpha: 1), itemCount: 6),
-        User(pk: 2, name: "Ryan Olson", handle: "jessie_squires"),
+        User(pk: 2, name: "Ryan Olson", handle: "ryanolsonk"),
         "Praesent commodo cursus magna, vel scelerisque nisl consectetur et.",
         User(pk: 4, name: "Oliver Rickard", handle: "ocrickard"),
         GridItem(color: UIColor(red: 56/255.0, green: 151/255.0, blue: 240/255.0, alpha: 1), itemCount: 5),
         "Nullam quis risus eget urna mollis ornare vel eu leo. Praesent commodo cursus magna, vel scelerisque nisl consectetur et.",
-        User(pk: 3, name: "Jessie Squires", handle: "ryanolson"),
+        User(pk: 3, name: "Jessie Squires", handle: "jesse_squires"),
```
Comment (jessesquires) [inline Example/IGListKitExamples/ViewControllers/MixedDataViewController.swift]: lol. missed this too.

s/Jessie/Jesse

Comment (rnystrom) [inline Example/IGListKitExamples/ViewControllers/MixedDataViewController.swift]: > Jazzy Squirrels

(i'll never get this right)

Comment (jessesquires) [inline Example/IGListKitExamples/ViewControllers/MixedDataViewController.swift]: LOL 😆

**Pull request closed without merging**